lib/Doctrine/ORM/Configuration.php

```diff
@@ -816,7 +824,7 @@ public function getSecondLevelCacheConfiguration()
     /**
      * Returns query hints, which will be applied to every query in application
      *
-     * @return array
+     * @psalm-return array<string, mixed>
```

Comment:
```suggestion
     * @psalm-return array<string, \Doctrine\ORM\Query::HINT_*>
```

Same as previous but for `\Doctrine\ORM\Query::HINT_*`